File: iniciante/swift/b1020.swift
import Foundation

let ageDays = Int(readLine()!)!

let year = ageDays / 365
let months = (ageDays % 365) / 30
let days = (ageDays % 365) % 30
    
print("\(year) ano(s)")
print("\(months) mes(es)")
print("\(days) dias(s)")

File: iniciante/javascript/b1020.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

let diasTotal = parseInt(lines.shift());
let ano = Math.floor(diasTotal / 365);
let meses = Math.floor((diasTotal % 365) / 30);
let dias = Math.floor((diasTotal % 365) % 30);
console.log(ano, 'ano(s)');
console.log(meses, 'mes(es)');
console.log(dias, 'dia(s)');
